pkg/scheduler/service/bookkeepingtask.go

```diff
@@ -0,0 +1,65 @@
+package service
+
+import (
+	"fmt"
+	"github.com/kyma-incubator/reconciler/pkg/model"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"time"
+)
+
+type BookkeepingTask interface {
+	Apply(*ReconciliationResult) []error
+}
+
+type orphanOperation struct {
```

Comment:
To make it's purpose more self-explaining, it could be worth to rename it to `markOrphanOperations` or similar.